gap-analysis/index.md

```diff
@@ -369,51 +369,52 @@ Such an attribute could be used by an authoring tool, for example, to encode the
 
 If (0,5) had another meaning, such as an open interval, or an ordered pair, or greatest common divisor, a value other than “point” would be used in the intent attribute.
 
+Here is some potential markup using `aria-label` for the remaining two examples.
+<details markdown="1">
+<summary><span markdown="1">Line segment example $\overline{A'B'}$ </span></summary>
 
-The line segment from the point A' to the point B' could be encoded as:
-
+This example shows the need to use nested `intent`s.
 ```xml
-<mover intent="segment($x,$y)">
+<mover intent="line-segment($x,$y)">
   <mrow>
-    <msup arg="x" intent="prime($1)">
+    <msup arg="x" intent="primed($1)">
```

Comment:
I still think this is wrong, primed, like prime is not giving any "intent" it's just describing the character used in the presentation. It's as if you had the ambiguous notation |x| and used intent="surrounded-by-vertical-bars" 

as the meaning here is simply an identifier and ' isn't being used as an operation on B I think something like intent="B-primed" would be more natural (or no intent at all)

Reply:
I've learned something today, indeed "primed" with a "d" is the accepted adjective to refer to these variable constructs in mathematical English. And yet everyone I have heard speak them does so with a trailing "-prime", no audible "d".

I think the natural suggestions are that it ought to be either `primed($1)` or `B-prime`. I lean to the latter myself, since I am afraid of extrapolations to `asterisked($1)` or `tilded($1)`, but feel happy with `B-asterisk` and `B-tilde`.

[Unrelated aside: My search for "primed" even revealed another [meaning of the prime](https://people.eecs.ku.edu/~bluo/download/bool1.pdf), to mean logical negation in one set of notes on Boolean Algebra. Fun.]

Reply:
B-prime is is OK with me (and how I'd pronounce it normally)